api/src/main/java/uk/gov/hmcts/bar/api/data/service/PaymentInstructionService.java

```diff
@@ -97,6 +98,11 @@ public PaymentInstruction createPaymentInstruction(BarUser barUser, PaymentInstr
         paymentInstruction.setUserId(barUser.getId());
         PaymentInstruction savedPaymentInstruction = paymentInstructionRepository.saveAndRefresh(paymentInstruction);
         savePaymentInstructionStatus(savedPaymentInstruction, barUser.getId());
+        if ((barUser.getRoles().contains(BarUserRoleEnum.BAR_FEE_CLERK.getIdamRole()))) {
+            savedPaymentInstruction.setStatus(PaymentStatusEnum.PENDING.dbKey());
```

Comment:
only this  line should be here. move the if block up few lines.

Reply:
We need to insert two status records for fee clerk role  - one for 'Draft' and one for 'Pending'.